### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by zzwen1** [lxtGH/PFSegNets#7]
For vaihingen and Potsdam dataset, how do you divide the training set and the test set

**Comment by lxtGH** [comment]
Please see the sublimentary part of the paper.
![image](https://user-images.githubusercontent.com/17202268/147520118-704f644c-f2d5-4444-bf96-1f12eaf6238e.png)

**Comment by zzwen1** [comment]
what is the validation
